tests/VObject/ITip/BrokerProcessReplyTest.php

```diff
@@ -373,6 +373,129 @@ public function testReplyNewExceptionTz(): void
         $this->process($itip, $old, $expected);
     }
 
+    public function testReplyNewExceptionReccurenceIdInDifferentTz(): void
+    {
+        // This is a reply to 1 instance of a recurring event. This should
+        // automatically create an exception.
```

Comment:
I don't understand why this comment has "This should automatically create an exception."

The source code now handles this situation. 
`TZID=Asia/Ho_Chi_Minh:20140725T110000` is the event recurrence of `DTSTART;TZID=America/Toronto:20140724T000000` that happens the next day, 20140725.

`America/Toronto` is UTC-4, `Asia/Ho_Chi_Minh` is UTC+7. So they are 11 hours apart. `0000` in Toronto is the same time as `1100` in Ho Chi Minh City.